Add Ritocoin (RITO)

Scotty0448 opened the pull request.

iancoleman commented:
Thanks for this.

The test is not passing for me, private key in the test is
L1CyVD5ADNgSUxZn6kRpJe9e17FDuAZzRGwNjvDBnEqRWjo4SEAX
but using the tool I get
Mdaumz3494kxCeiEBame4ZBzjtTQ5mYzD8notv2EBW8FcNy3PiYd

When I put both private keys into
https://wallet.ritocoin.org/#verify
they give the same address (matching the test).

Any thoughts?

Scotty0448 commented:
Oh, when I corrected the wif value in the second commit I forgot to update the test. The firstPrivKey in the test should now be Mdaumz3494kxCeiEBame4ZBzjtTQ5mYzD8notv2EBW8FcNy3PiYd. You'll need to make the change now because I already deleted my branch when I saw you merge the pull request.

iancoleman commented:
No worries I've made that change, it'll be pushed soon. Thanks again!
